### DISCUSSION TITLE
Dark mode not saved

### DISCUSSION THREAD

**Discussion by yossile** [openscopeproject/TrguiNG#177] [Q&A] [answered]
Hi, I have transmission-daemon serving the UI.
When I'm switching in the browser to dark mode and hitting the save button I see a popup saying it is saved but I then refresh the page and it's back to light theme.
I have the latest release.

**Comment by yossile** [comment]
Ok, I clicked on the lower-right moon icon (not in settings) and it actually saved the dark mode theme even after refresh.
Not sure why the settings option isn't being save though. I can open an issue but the alternative is fine for me.

**Comment by qu1ck** [comment, marked as answer]
Fixed in 09ccc6a
